packages/xl-multi-column/src/extensions/DropCursor/MultiColumnDropCursorPlugin.ts

```diff
@@ -267,20 +274,22 @@ class DropCursorView {
         ) {
           const block = this.editorView.nodeDOM(this.cursorPos.pos);
 
-          const blockRect = (block as HTMLElement).getBoundingClientRect();
-          const halfWidth = (this.width / 2) * scaleY;
-          const left =
-            this.cursorPos.position === "left"
-              ? blockRect.left
-              : blockRect.right;
-          rect = {
-            left: left - halfWidth,
-            right: left + halfWidth,
-            top: blockRect.top,
-            bottom: blockRect.bottom,
-            // left: blockRect.left,
-            // right: blockRect.right,
-          };
+          if (block !== null) {
```

Comment:
does this actually fix the issue or only prevents the error? Maybe discuss this together?

Reply:
fixed